src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
+@import "./reset";
+@import "./_variables";
+
 body {
-  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $calendar-padding;
+  width: $calendar-size;
+  gap: $cell-gap;
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $cell-bg-color;
+    transition: all 0.3s;
+
+    @media (any-hover: hover) {
+      &:hover {
+        transform: translateY(-20px);
+        background-color: $cell-bg-color-hover;
+      }
+    }
```

Comment:
👍  for digging into a hover media query

However, in the current task, you don't need to use it, as the hover effect does not affect the usability on non-hover devices

No need to fix this - just keep in mind for the next time